website/addons/mendeley/static/user-cfg.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+
+var $ = require('jquery');
+var ko = require('knockout');
+var $osf = require('osfHelpers');
+
+var MendeleyAccount = function(display_name, id) {
+    var self = this;
+    self.display_name = display_name;
+    self.id = id;
+};
+
+var MendeleyUserSettingsViewModel = function() {
+    var self = this;
+    self.accounts = ko.observableArray();
+
+    self.updateAccounts = function() {
+        var request = $.ajax({
+            url: '/api/v1/settings/mendeley/accounts/'
+        });
+        request.done(function(data) {
+            self.accounts([]);
+            ko.utils.arrayMap(data.accounts, function(acct) {
+                self.accounts.push(
+                    new MendeleyAccount(acct.display_name, acct.id)
+                )
+            })
+        });
+        request.fail(function() {
+            console.log('fail');
+        });
+    };
+
+    self.connectAccount = function() {
+        window.oauth_complete = function() {
+            self.updateAccounts();
+        };
+        window.open('/oauth/connect/mendeley/');
+    };
+
+    self.disconnectAccount = function(account) {
```

Comment:
We usually show a confirmation dialog before deleting anything. Can we add one here as well?